### DISCUSSION TITLE
Large PDN pitch

### DISCUSSION THREAD

**Discussion by maliberty** [The-OpenROAD-Project/OpenLane#1239] [Q&A]
I see in the OL configuration

set ::env(FP_PDN_VPITCH) 153.6
set ::env(FP_PDN_HPITCH) 153.18

In ORFS we use

add_pdn_stripe -grid {grid} -layer {met4} -width {1.600} -pitch {27.140} -offset {13.570}
add_pdn_stripe -grid {grid} -layer {met5} -width {1.600} -pitch {27.200} -offset {13.600}

These values are substantially different.  I am concerned that the OL values are quite large.  Has there been verification of IR drop with these OL settings?  @shalan

**Comment by QuantamHD** [comment]
Hmm, that does seem substantial. Seems like we should generally be a bit more conservative in the PDN grid given many users are not checking their IR drop with a lot of rigor.

**Comment by shalan** [comment]
@maliberty @QuantamHD. These were set a while back based on a back-of-the-envelope calculation. If I recall, this would result in ~2 Ohms per a strap segment.

Let me ask the same question. How did you come up with ORFS values? They seem to be too aggressive as the PDN would consume routing resources on m4 & m5.

Btw, by default OL generates a report for the IR drops (pdn.pga.rpt). The designer should adjust these parameters based on values shown in the report.

**Reply by maliberty** [reply]
I believe they were based on some IR drop analysis done by students.  I would match ORFS make OL if I had confidence in its correctness.

What is your definition of a "strap segment".  Would you provide the calculations that you base this on.
